examples/04_local_pickup/03_get_preferred_pickup_location_availability/query.graphql

```diff
@@ -0,0 +1,20 @@
+# Before sending this request, please make sure your app has unauthenticated_read_product_pickup_locations scope.
+# This query will return the availability of the product of a specific location. You can pass a location ID using preferredLocationId argument to quert the location.
+
+query GetPreferredStoreAvailability @inContext(preferredLocationId: "gid://shopify/Location/65607794710") {
+  product(handle: "Orange") {
+    variantBySelectedOptions(selectedOptions: {name: "Weight", value: "1 lb"}) {
+      storeAvailability(first: 1) {
+        edges {
+          node {
+            available
+            pickUpTime
+            location {
+              name
+            }
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
[Similar comment to what I said here](https://github.com/Shopify/storefront-api-learning-kit/pull/26/files#r1086166696)

Reply:
variables.json file added for getPreferredStoreAvailability query